jest.config.js

```diff
@@ -49,7 +49,9 @@ module.exports = {
       'plugin-simple-throttle',
       'plugin-console-breadcrumbs',
       'plugin-browser-session'
-    ]),
+    ], {
+      setupFiles: ['<rootDir>/jest/setup/mockEventTarget.js']
```

Comment:
See https://github.com/jsdom/jsdom/issues/2156

This is needed to fix the `TypeError: 'addEventListener' called on an object that is not a valid instance of EventTarget.` seen in the integration tests following the rollup changes